libs/ngxtension/computed-from/src/computed-from.spec.ts

```diff
@@ -0,0 +1,203 @@
+      TestBed.runInInjectionContext(() => {
+        const value = new BehaviorSubject(1);
+        const s = computedFrom({value});
+        expect(s()).toEqual({value: 1});
+      });
+    });
+  });
+  describe('works with observables and signals', () => {
+    it('value inside array', () => {
+      TestBed.runInInjectionContext(() => {
+        const valueS = signal(1);
+        const valueO = new BehaviorSubject(1);
+        const s = computedFrom([valueS, valueO]);
+        expect(s()).toEqual([1, 1]);
+      });
+    });
+    it('value inside object', () => {
+      TestBed.runInInjectionContext(() => {
+        const valueS = signal(1);
+        const valueO = new BehaviorSubject(1);
+        const s = computedFrom({valueS, valueO});
+        expect(s()).toEqual({valueS: 1, valueO: 1});
+      });
+    });
+  });
+  describe('works with observables, signals and rxjs operators', () => {
+    it('by using rxjs operators directly', () => {
+      TestBed.runInInjectionContext(() => {
+        const valueS = signal(1);
+        const valueO = new BehaviorSubject(1);
+
+        const s = computedFrom(
+          [valueS, valueO],
+          map(([s, o]) => [s + 1, o + 1])
+        );
+
+        expect(s()).toEqual([2, 2]);
+      });
+    });
+    it('by using pipe operator', () => {
+      TestBed.runInInjectionContext(() => {
+        const valueS = signal(1);
+        const valueO = new BehaviorSubject(1);
+
+        const s = computedFrom(
+          [valueS, valueO],
+          pipe(
+            map(([s, o]) => [s + 1, o + 1]),
+            filter(([s, o]) => s === 2 && o === 2)
+          )
+        );
+
+        expect(s()).toEqual([2, 2]);
+
+        valueS.set(2);
+        valueO.next(2);
+
+        expect(s()).toEqual([2, 2]);
+      });
+    });
+
+    describe('by using async operators', () => {
+      @Component({standalone: true, template: '{{s()}}', imports: [JsonPipe]})
+      class TestComponent {
+        valueS = signal(1);
+        valueO = new BehaviorSubject(1);
+
+        s = computedFrom(
+          [this.valueS, this.valueO],
+          pipe(
+            map(([s, o]) => [s + 1, o + 1]),
+            switchMap((v) => of(v).pipe(delay(1000)))
+          )
+        );
+      }
+
+      let component: TestComponent;
+      let fixture: ComponentFixture<TestComponent>;
+
+      beforeEach(async () => {
+        await TestBed.configureTestingModule({imports: [TestComponent]}).compileComponents();
+        fixture = TestBed.createComponent(TestComponent);
+        component = fixture.componentInstance;
+      });
+
+      it('should handle async stuff', fakeAsync(() => {
+        fixture.detectChanges(); // initial change detection to trigger effect scheduler
+        expect(component.s()).toEqual([1, 1]); // initial value is 1,1 because of delay in switchMap
+        expect(fixture.nativeElement.textContent).toEqual('1,1');
+
+        fixture.detectChanges(); // trigger effect scheduler (for the moment)
```

Comment:
We will be able to use `Testbed.flushEffects()` from this PR https://github.com/angular/angular/pull/51049